spec/controllers/people_controller_spec.rb

```diff
@@ -11,15 +11,15 @@
     it 'should save nationality2 as nil if checkbox is unchecked and save it if it is checked' do
       expect(person["nationality2"]).not_to be(nil)
       put :update, params: {id: person["id"], person: person, has_nationality2: {checked: "0"}}
-      response.code.should == "302"
-      response.should redirect_to(person_path(person["id"]))
+      expect(response.code).to include("302")
```

Comment:
Hmm I think the way it was in the first place was correct. We expect response code `302` nothing else. Although not realistic in this case `3020` would also match